support-frontend/assets/helpers/tracking/ophan.ts

```diff
@@ -139,6 +139,7 @@ const navigateWithPageView = (
 	navigate: NavigateFunction,
 	destination: string,
 	participations?: Participations,
+	replace?: boolean,
```

Comment:
Maybe instead of the `replace` parameter we have an optional `options` parameter, which is an object which matches the model of the `options` defined here: https://reactrouter.com/en/main/hooks/use-navigate#usenavigate? This would be more flexible if in the future we wish to use any additional options.

Reply:
Yes, even better , code looks cleaner too, done.